### PR TITLE
Removed some old ava stuff

### PR DESCRIPTION
**Pull request by im-martijn**
- Instructions was the old one from the ava workshop
- Removed old line refering to nyc in .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 node_modules/
 coverage/
-.nyc_output/
```
Comment (kentcdodds) [inline .gitignore:3]: This is great :)

File: INSTRUCTIONS.md [hunk 2]
```diff
@@ -1,627 +0,0 @@
...
-a few more dependencies. Oh, and one more thing, let me introduce you to your new
-best friend:
-
-🐯 *- Hi! I'm Terry the Tiger! These instructions are really long and boring! So*
-*I'll pop up here and there where you'll be expected to actually do something!*
-*And if you really want to skip around, just copy me and <kbd>⌘</kbd>+<kbd>f</kbd>*
-*(or <kbd>CTRL</kbd>+<kbd>f</kbd> on windows) for me on the page. See you around!*
-
-# Starting here, things are old. I'm migrating this from AVA
-# If you wanna help, that'd be great
```
Comment (kentcdodds) [inline INSTRUCTIONS.md:28]: Note this. I'd actually rather keep things around as a reference for when I finally do update the instructions. If you want to do that, be my guest! :D

**Review by kentcdodds — CHANGES_REQUESTED** [review]

File: INSTRUCTIONS.md (hunk 2 above)
Comment (im-martijn) [reply INSTRUCTIONS.md:28]: I'm not sure I'd be able to. Not much experience with either Jest or Ava. That's why I was looking at this repo. ;)

Comment (kentcdodds) [reply INSTRUCTIONS.md:28]: That's fine. Thanks anyway.

**Comment by im-martijn** [comment]
I restored INSTRUCTIONS.md.

**Comment by kentcdodds** [comment]
Make sure that you force push that change. The PR still shows the instructions removed.

**Comment by im-martijn** [comment]
Are you sure? I see the [commit from October 31st](https://github.com/kentcdodds/react-jest-workshop/pull/3/commits/a1d14b6cfbd00cdd55d417672209b0fc8fafe5fd), which adds back the INSTRUCTIONS.md. Or am I wrong?

**Comment by kentcdodds** [comment]
Whoops, you're correct! Thank you!

**Pull request merged**